Explore: Download traces as JSON in Explore Inspector (#38614)
grafana/grafana: docs/sources/explore/explore-inspector.md

1. Inspect the log query results as described above.
1. Click **Download logs**.

### Download trace results

> Grafana generates a JSON file for the trace results based on the trace datasource type. Jaeger, Zipkin, and OTLP formats are supported.

Reviewer: ```suggestion
Based on the data source type, Grafana generates a JSON file for the trace results in one of the supported formats: Jaeger, Zipkin, or OTLP formats.
```

1. Open the inspector.

> 1. Inspect the trace query results as described above.

Reviewer: Link to #inspect-raw-query-results.
